src/utils/baseUrl.ts

```diff
@@ -10,13 +10,7 @@ const getBaseUrl = () => {
   }
 
   // Otherwise, we're probably running locally?
-  if (process.env.NODE_ENV === 'production') {
-    return 'https://gallery.so';
-  }
-
-  // TODO: figure out if/what hostname needed for tests
-
-  return `http://localhost:3000`;
+  return 'http://localhost:3000';
```

Comment:
This assumes that if there's no vercel env var, then we're most likely on localhost, instead of assuming we're on the production url